Client/src/index.css

```diff
@@ -69,10 +69,8 @@ a:hover {
 
 body {
   margin: 0;
-  display: flex;
-  place-items: center;
   min-width: 320px;
-  min-height: 100vh;
+  height: 100%;
```

Comment:
Does this behave correclty if there is not enough content to fill the screen?

I assume the `min-height: 100vh` was set to make sure the body always at least fills the screen, even if there is no content.

Reply:
Well it's more of a redundant line. The sidebar is set to `height: calc(100vh - 64px)`  64 pixels being the height of the navBar (will need to add a media query to handle navBar height changes) and the main-content which wraps the sidebar has a height of `max-content` so automatically the main-content div will fill the remainder of the screen + overflow if there is any.